pytorch_toolkit/tools/instantiate.py

```diff
@@ -14,23 +14,30 @@
 
 import argparse
 import glob
+import logging
 import os
 from subprocess import run
 
 import yaml
 
+def run_with_log(*args, **kwargs):
+    cmd = args[0]
+    logging.info(f'Running command\n`{cmd}`') #TODO: consider with Ilya
```

Comment:
I'm ok, remove comment, but could you move move it to 'ote' package somewhere and use in similar cases

Reply:
Ilya, I checked -- there are similar functions in couple of small scripts (like `tools/instantiate.py`) that do not depend on `ote` module or any other custom python submodule from OTE. 
I do not want to add dependencies in these scripts due to such small function.
So, I'd prefer to keep them as is.

Reply:
ok